src/store/ChartStore.ts

```diff
@@ -1246,9 +1246,11 @@ class ChartStore {
         const fncToCall = fullscreen_map[isInFullScreen ? 'fnc_exit' : 'fnc_enter'].find(
             fnc => (el as HTMLElement)[fnc as keyof HTMLElement]
         );
-
+        // fncToCall can be undefined for iOS' Safari that does not support fullscreenAPI
         if (fncToCall) {
-            (el as HTMLElement)[fncToCall as 'requestFullscreen']();
+            (el as HTMLElement)[fncToCall as 'requestFullscreen']()?.catch(() => {
```

Comment:
'?' is needed to avoid 'TypeError: undefined is not an object (evaluating 'el[fncToCall]()["catch"]')' error.